personal_assistant/pa_tag/views.py

```diff
@@ -0,0 +1,24 @@
+from django.shortcuts import render, redirect
+from .models import Tag
+from .forms import TagForm
+
+def main(request):
+    tags =  Tag.objects.all()
+    context = {"tags":tags}
+
+    return render(request, 'tags/tags.html', context)
```

Comment:
Назвати папку як і апку:
```suggestion
    return render(request, 'pa_tag/overview.html', context)
```